src/Aspire.Hosting.Azure.SignalR/Aspire.Hosting.Azure.SignalR.csproj

```diff
@@ -11,11 +11,13 @@
   <PropertyGroup>
     <MinCodeCoverage>96</MinCodeCoverage>
   </PropertyGroup>
-
+  
   <ItemGroup>
     <ProjectReference Include="..\Aspire.Hosting.Azure\Aspire.Hosting.Azure.csproj" />
     <PackageReference Include="Azure.Provisioning" />
     <PackageReference Include="Azure.Provisioning.SignalR" />
+    <PackageReference Include="Microsoft.Azure.SignalR.Management" />
+    <PackageReference Include="Microsoft.AspNetCore.SignalR.Client" />
```

Comment:
Why do you need the signalr client reference? Looks like it's only used in a test, so shouldn't the reference be on the test project?

Reply:
you're right, let me move it to test project